src/autosuggest/internal.tsx

```diff
@@ -202,6 +201,7 @@ const InternalAutosuggest = React.forwardRef((props: InternalAutosuggestProps, r
       ariaLabel={ariaLabel}
       ariaRequired={ariaRequired}
       clearAriaLabel={clearAriaLabel}
+      isEmpty={isEmpty && !dropdownStatus.content}
```

Comment:
An alternative way to not show the dropdown is to pass null as dropdownContent. This approach is used in the property-filter already.

See https://github.com/cloudscape-design/components/blob/main/src/internal/components/autosuggest-input/index.tsx#L282

Reply:
Also take a look at potentially related PR: https://github.com/cloudscape-design/components/pull/1574. The autosuggest-input considers the presence of content but not footer. However, the empty state comes from the footer apparently.